styleguide/source/assets/scss/03-organisms/_info-card-group.scss

```diff
@@ -0,0 +1,8 @@
+.ama__info-card-group {
```

Comment:
Is this class being used by anything? I can't seem to find the corresponding twig file that has the the class ama__info-card-group in it.

Reply:
Oh it is not. This has been removed. I totally forgot the CSS thanks for flagging!